src/main/app/src/app/taken/taak-view/taak-view.component.ts

```diff
@@ -212,8 +214,10 @@ export class TaakViewComponent extends AbstractView implements OnInit, AfterView
         return this.ingelogdeMedewerker.gebruikersnaam !== this.taak.behandelaar?.gebruikersnaam;
     }
 
-    assignToMe(event: any): void {
+    assignToMe(): void {
+        this.websocketService.suspendListener(this.taakListener);
         this.takenService.assignToLoggedOnUser(this.taak).subscribe(() => {
+            this.taak.behandelaar = this.ingelogdeMedewerker;
```

Comment:
Ik heb in de assignToLoggedOnUser methode in de takenService gezorgd dat er een taak teruggegeven wordt, zodat we niet meer de ingelogdeMedewerker hoeven te setten op de zaak/taak bij assignToMe. Dus even de behandelaar uit de teruggegeven taak halen en die setten als behandelaar.